WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -298,9 +298,16 @@ extension WPRichContentView: WPTextAttachmentManagerDelegate {
         let finalSize = efficientImageSize(with: url, proposedSize: proposedSize)
         let image = richTextImage(with: finalSize, url, attachment)
 
-        // show that something is loading.
-        attachment.maxSize = CGSize(width: proposedSize.width, height: proposedSize.height)
+        let needsLayoutRefreshWhenLoaded = finalSize.height == 0
 
+        // show that something is loading.
+        if finalSize.height == 0 {
```

Comment:
Could this be `if needsLayoutRefreshWhenLoaded ` ?

Reply:
Good call! I don't love the variable name for that though, because the name is tightly associated with the `layoutAttachmentViews` call.

What about renaming `needsLayoutRefreshWhenLoaded ` to `isUsingTemporaryLayoutDimensions`? Then we can use it in both places and it should be logical?

Reply:
sounds good to me!

Reply:
Sweet – done!